### PR TITLE
[iExplorer] Update version and url

### PR DESCRIPTION
**Pull request by n0ts**
*If there’s a checkbox you can’t complete for any reason, that's okay, just explain in detail why you weren’t able to do so.*

After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name and version.

### CODE REVIEW

File: Casks/iexplorer.rb [hunk 1]
```diff
@@ -1,8 +1,8 @@
 cask 'iexplorer' do
-  version '4.0.11.0'
+  version '4.0.11'
   sha256 'b9d222d53cdcc53f0137140d48ab01ff782860c9872397c7462d23f6dbedcb8b'
 
-  url "https://www.macroplant.com/release/iExplorer-#{version}.dmg"
+  url "https://assets.macroplant.com/downloads/iExplorer-#{version}.dmg"
   appcast "https://www.macroplant.com/iexplorer/ie#{version.major}-appcast.xml",
           checkpoint: '48eaa402b874473b322a88d0af31c3642f1c2588a3337524a95a71c2d5fd564d'
```
Comment (miccal) [inline Casks/iexplorer.rb]: You also need to update the [`checkpoint`](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/appcast.md).

Comment (n0ts) [inline Casks/iexplorer.rb]: iExplorer appcast is not changed from https://www.macroplant.com/iexplorer/ie4-appcast.xml

**Comment by n0ts** [comment]
@miccal Oh... Thanks a lot!

**Comment by miccal** [comment]
Cannot seem to find a stable `checkpoint` for the `appcast`.

**Pull request merged**